bundle.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=cert-manager-helm-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```

Comment:
It is configured an alpha release, but the release that will be created by the github action once this PR gets merged is a stable, right?

Maybe the prepare-stable-release target should be run?